File: Lista_Geral_Python/comandos de entrada e saida/15question.py
### Escreva um programa que leia uma palavra e um número inteiro k e identifique a k-
# ésima letra da palavra informada pelo usuário.

palavra = str(input("Insira uma palavra: "))
numero = int(input("Informe um número: "))
if numero > len(palavra):  
    print("Valor informado e maior que a palavra.")
else:
    print(f"A {numero}º letra da palavra: {palavra} é {palavra[numero-1]}")

File: comandos de entrada e saida/15question.c
// Escreva um programa que leia uma palavra e um número inteiro k e identifique a k-
// ésima letra da palavra informada pelo usuário.
#include <stdio.h>

int main(){

    char palavra[30], letra;
    int numero;

    printf("Informe uma palavra: ");
    scanf("%s", palavra);
    printf("Informe um numero: ");
    scanf("%d", &numero);
    letra = palavra[numero-1];
    printf("A letra na posicao %d da palavra %s e: %c", numero, palavra, letra);

    return 0;
}